base-helm-configs/prometheus-snmp-exporter/values.yaml

```diff
@@ -240,7 +240,7 @@ serviceMonitor:
   ## If true, a ServiceMonitor CRD is created for snmp-exporter itself
   ##
   selfMonitor:
-    enabled: false
+    enabled: true
```

Comment:
Generally don't need this unless you're trying to collect metrics about the exporter service itself. re; if you're running into issues with the exporter and want to gain some insight into why the exporter isn't behaving or performing as expected. The metrics collected here are unrelated to snmp metrics you're configuring to collect

Reply:
This is just a nit really.. it's a little extra overhead for not a lot of insight but not a blocker. Feel free to resolve without changes